index.js

```diff
@@ -0,0 +1,21 @@
+import {
+  displayList, displayBookForm, displayContact, listBtn, addBtn1, contactBtn,
+} from './modules/menuClass.js';
+
+import { clockDisplay } from './modules/variables.js';
+
+import { DateTime } from './modules/luxon.js';
+
+listBtn.addEventListener('click', displayList);
+
+addBtn1.addEventListener('click', displayBookForm);
+
+contactBtn.addEventListener('click', displayContact);
+
+const timer = () => {
+  const today = DateTime.now();
+  clockDisplay.innerText = `${today.toLocaleString(DateTime.DATETIME_FULL)}`;
+};
+
+timer();
+setInterval(() => { timer(); }, 1000);
```

Comment:
- [ ] (Optional) Since you have declared `timer`, you can just call it as a callback function when using the `setInterval` method. `setInterval(timer, 1000)`. 👍